frontend/packages/dev-console/src/components/topology/TopologyDataController.tsx

```diff
@@ -93,6 +100,46 @@ export const TopologyDataController: React.FC<TopologyDataControllerProps> = ({
       optional: true,
     });
   }
+
+  React.useEffect(() => {
+    let ignore = false;
+
+    const fetchHelmReleases = async () => {
+      let releases: HelmRelease[];
+      try {
+        releases = await coFetchJSON(`/api/helm/releases?ns=${namespace}`);
+      } catch {
+        return;
+      }
+      if (ignore) return;
+
+      const releaseResourcesMap = releases.reduce((acc, release) => {
+        const manifestResources: K8sResourceKind[] = safeLoadAll(release.manifest);
+
+        manifestResources.forEach((resource) => {
+          const resourceKindName = `${resource.kind}---${resource.metadata.name}`;
+          if (!acc.hasOwnProperty(resourceKindName)) {
+            acc[resourceKindName] = {
+              releaseName: release.name,
+              chartIcon: release.chart.metadata.icon,
```

Comment:
we didn't have this capability to show icons for each chart that's why we showed helm icon for all the helm release groups. Now that we are getting the icon for each chart, we should be showing the chart icon for releases instead of helm icon. Created a bug to address this https://issues.redhat.com/browse/ODC-3154